### ISSUE TITLE
More options for conditional chips

### ISSUE THREAD

**Issue by kineticscreen** [piitaya/lovelace-mushroom#154]
Would be good to have some more options for Conditional Chips - i.e. < > <! !>

Is / is not works for lights and switches, but not for sensors.

**Comment by piitaya** [comment]
As a workaround for now, your can define a [`binary_sensor` with template](https://www.home-assistant.io/integrations/template/#state-based-template-sensors) and use it in your condition 🙂

**Comment by kineticscreen** [comment]
That's... quite clever.

**Comment by KrX3D** [comment]
Hi, i would also like to use within the conditional chip a custom card

i.e.

i use this card on my dashboard:

`

type: conditional
conditions:
  - entity: sensor.seventeentrack_packages_in_transit
    state_not: '0'
card:
  type: custom:seventeen-track-card
  entity: sensor.seventeentrack_packages_in_transit
  title: Unterwegs
  location: false`

but:

`type: custom:mushroom-chips-card
chips:
  - type: conditional
    conditions:
      - entity: sensor.seventeentrack_packages_in_transit
        state_not: '0'
    chip:
      type: custom:seventeen-track-card
      entity: sensor.seventeentrack_packages_in_transit
      title: Unterwegs
      location: false
`

doesnt show anything

**Comment by janstadt** [comment]
Trying to think of a way to template out a conditional chip that displays whether or not the current user is a specific person. Would anyone be able to show me how to template out an is_user binary_sensor using the `{{user}}` variable?

**Comment by davidcoulson** [comment]
Use this card instead?

https://github.com/thomasloven/lovelace-state-switch

**Comment by janstadt** [comment]
> Use this card instead?
> 
> https://github.com/thomasloven/lovelace-state-switch

Thanks for the reply @davidcoulson. Im just trying to implement what @piitaya mentioned above for my specific use case.

**Comment by janstadt** [comment]
I tried doing a conditional chip but the alignment was all out of wack. Ended up with this which is more verbose but works. Leaving it here in case anyone else needs an example:

```
type: custom:state-switch
entity: user
default: default
states:
  Jake:
    type: custom:mushroom-chips-card
    chips:
      - type: weather
        entity: weather.home
        show_conditions: true
        show_temperature: true
      - type: alarm-control-panel
        entity: alarm_control_panel.ha_alarm
      - type: entity
        entity: sensor.exchange_rate_1_btc
        icon: mdi:bitcoin
        icon_color: orange
    alignment: center
  default:
    type: custom:mushroom-chips-card
    chips:
      - type: weather
        entity: weather.home
        show_conditions: true
        show_temperature: true
      - type: alarm-control-panel
        entity: alarm_control_panel.ha_alarm
    alignment: center
```

**Comment by piitaya** [comment]
Thanks for sharing. 
The only solution is to create a condition template chip. We works on other cards for now but that something we can consider after 😉

**Comment by marcusforsberg** [comment]
> The only solution is to create a condition template chip. We works on other cards for now but that something we can consider 😉

This would be absolutely awesome to be able to create advanced conditionals without resorting to binary sensors. I think it would be fine without a UI for more advanced users. 😊

**Comment by SaSa1983** [comment]
> Thanks for sharing. The only solution is to create a condition template chip. We works on other cards for now but that something we can consider 😉 Templates are not user friendly but allow maximum flexibility. Building a condition builder in the UI will be to complicated.

Actually there's no need to build a complex editor:

Currently you can add multiple (and connected) conditions based on entities.
You only need to add a "select" for condition type:

* Entity: should be the default in order for "migration" and displays the current input fields
* Template: hides the entity input fields ans shows a single "text" input allowing to input template syntax, whereas the expression must evaluate to boolean, e.g. "expand(states.update) | selectattr('state', 'eq', 'on') | list | length > 0" to know if any update is available

**Comment by diginfo** [comment]
Would it be possible to use the state or attribute of a sensor as the condition:

```
      - type: conditional
        conditions:
          - entity: sensor.battery_device_a
            operator: "<"
            state: {{ input_number.low_batt_level  }}

```

I want to show devices where the battery level is less than a input_number slider

**Comment by orimate** [comment]
Now It works as I want, but a conditional template with operators would be really nice.

My config would be better if sensor (sensor.netatmo_co2_indoor_mainunit_0) would be declared as a variable over all.

```
type: custom:mushroom-chips-card
chips:
  - type: template
    icon: |-         
      {% if states('sensor.netatmo_co2_indoor_mainunit_0')|float >= 1000 %}
      mdi:molecule-co2
      {% else %}
        
      {% endif %}     
    icon_color: |-
      {% if states('sensor.netatmo_co2_indoor_mainunit_0')|float >= 1800 %}
      #ff471a
      {% elif states('sensor.netatmo_co2_indoor_mainunit_0')|float >= 1000 %}
      #ff9900
      {% else %}

      {% endif %}
    content: |-
      {% if states('sensor.netatmo_co2_indoor_mainunit_0')|float >=1000  %}
      {{ states('sensor.netatmo_co2_indoor_mainunit_0') }} ppm
      {% else %}
        
      {% endif %} 
    card_mod:
      style: |-
        {% if states('sensor.netatmo_co2_indoor_mainunit_0')|float >= 1000 %}
        ha-card {
        
        }
        {% else %}
        ha-card {
          --chip-box-shadow: none;
          --chip-background: none;                
        }
        {% endif %} 
```

**Comment by piitaya** [comment]
@orimate You can use entity variable (https://github.com/piitaya/lovelace-mushroom/blob/main/docs/cards/template.md#description) :


```yaml
- type: template
  entity: sensor.netatmo_co2_indoor_mainunit_0
  icon: |-         
    {% if states(entity) | float >= 1000 %}
    mdi:molecule-co2
    {% else %}
      
    {% endif %}
```

**Comment by piitaya** [comment]
For now, there is no way to use template in condition because there is performance issue with template so I prefer to keep template for template card and template chip which are more advanced chips.

**Comment by MDausch** [comment]
I had poked around with this locally, and have been running a totally hacked together implementation for a while. I have it set up such that if the template returns 1, the desired chip shows, otherwise it will be hidden. Essentially just an inline binary_sensor.

This was really nice for quick one offs that I hadn't wanted to make a new template sensor for, but now that we have the ability to define template helpers in the gui I had started converting my conditions to them. Maybe there are some other nice to haves that others can think of that would make this a more appealing option still?

**Comment by piitaya** [comment]
**🚨 Spoiler alert 🚨** 

It's introduced with this PR https://github.com/piitaya/lovelace-mushroom/pull/1269. It will be available with `2023.11` Home Assistant release and `3.2.0` Mushroom release.

![image](https://github.com/piitaya/lovelace-mushroom/assets/5878303/f663186c-1e95-404d-b5e1-412b0ea9a1f2)

**Comment by KrX3D** [comment]
hi, is there any plan to use for chip the type card?

like i mentioned above, i would like to use a custom card
